lib/tasks/db.rake

```diff
@@ -0,0 +1,16 @@
+namespace :db do
+  desc "Add constraint to votes on topics. Vote >= 0"
+  task add_constraints: :environment do
+    ActiveRecord::Base.connection.execute <<-SQL
+      ALTER TABLE topics ADD CONSTRAINT positive_votes CHECK (votes >= 0);
```

Comment:
Apparently postgresql does not have unsigned integers, so i creted this rake task that extrends ´rake db` with `rake db: add_constraints`and `rake db: remove_constraints`. This rake task adds validation in the database so i hope that this ends up being atomic.